content/shapes/functions.ts

```diff
@@ -629,9 +1607,11 @@ function handlePathing(path: GeoPath, base: GeoPath, heightPath: GeoPath, height
   const pathAngle = pathSegment.angle % Math.PI;
   const perpendicular = pathAngle > correctAngle - angleRange && pathAngle < correctAngle + angleRange;
   if (perpendicular) {
-    path.$el.css({color: 'red'});
+    path.$el.addClass('angle-perpendicular');
+    path.$el.removeClass('angle-other');
   } else {
-    path.$el.css({color: 'blue'});
+    path.$el.addClass('angle-other');
+    path.$el.removeClass('angle-perpendicular');
```

Comment:
Rather than if/else, you could use `path.$el.setClass('angle-other', perpendicular)` for this. Is the second argument is truth, it adds the class, otherwise it removes it.

Reply:
[TODO]